libmscore/stafftextbase.cpp

```diff
@@ -63,6 +63,7 @@ void StaffTextBase::write(XmlWriter& xml) const
             int swingRatio = swingParameters()->swingRatio;
             xml.tagE(QString("swing unit=\"%1\" ratio= \"%2\"").arg(swingUnit).arg(swingRatio));
             }
+      xml.tagE(QString("capo fretId=\"%1\"").arg(capo()));
```

Comment:
Could we store the capo only if the fret is != 0 ? or that wouldn't work ?

Reply:
yes, that would work fine, just an oversight on my part :)